tests/unit/test_column_noise.py

```diff
@@ -65,9 +65,9 @@ def dummy_dataset():
 
     # Add a string_series column of mixed letters and numbers
     string_list = [
-        "foo1",
-        "bar2",
-        "baz3",
+        "fo1",
+        "fo2",
+        "fo3",
```

Comment:
Why were these changed?

Reply:
Just to test another variation of string formatting for test numerics.